src/server/plugins/nunjucks/context.js

```diff
@@ -41,6 +51,13 @@ export function context(request) {
     serviceName: config.get('serviceName'),
     serviceVersion: config.get('serviceVersion'),
     slug: params?.slug,
+    cookieConsent,
+    cookieConsentUpdated,
+    googleAnalyticsTrackingId: config.get('googleAnalyticsTrackingId'),
+    cspNonce: request?.plugins.blankie?.nonces?.script,
+    currentPath: request
+      ? `${request.path.toString()}${request.url.search}`
```

Comment:
Was ESLint kicking off that a string needed `.toString()`?

```suggestion
      ? `${request.path}${request.url.search}`
```

Reply:
Leftover code from my testing, thanks!